internal/kibana/synthetics/acc_test.go

```diff
@@ -0,0 +1,318 @@
+package synthetics_test
+
+import (
+	"testing"
+
+	"github.com/elastic/terraform-provider-elasticstack/internal/acctest"
+	"github.com/elastic/terraform-provider-elasticstack/internal/versionutils"
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+var (
+	minKibanaVersion = version.Must(version.NewVersion("8.14.0"))
+)
+
+const (
+	httpMonitorId = "elasticstack_kibana_synthetics_monitor.http-monitor"
+	tcpMonitorId  = "elasticstack_kibana_synthetics_monitor.tcp-monitor"
+
+	providerConfig = `
+provider "elasticstack" {
+  	elasticsearch {}
+	kibana {}
+	fleet{}
+}
+`
+
+	privateLocationConfig = `
+
+resource "elasticstack_fleet_agent_policy" "test" {
+	name            = "TestMonitorResource Agent Policy - test"
+	namespace       = "default"
+	description     = "TestMonitorResource Agent Policy"
+	monitor_logs    = true
+	monitor_metrics = true
+	skip_destroy    = false
+}
+
+resource "elasticstack_kibana_synthetics_private_location" "test" {
+	label = "TestMonitorResource-label"
+	space_id = "default"
+	agent_policy_id = elasticstack_fleet_agent_policy.test.policy_id
+}
+
+`
+
+	httpMonitorConfig = `
+
+resource "elasticstack_kibana_synthetics_monitor" "http-monitor" {
+	name = "TestHttpMonitorResource"
+	space_id = "default"
```

Comment:
Are you able to verify the resource behaviour (specifically imports) when the monitor is in a non-default namespace? 

In other [resources](https://github.com/elastic/terraform-provider-elasticstack/blob/main/internal/kibana/data_view/schema.go#L284) we encode both an ID (whatever that means for the specific resource) and space into the TF resource ID attribute. Originally this was to support importing since that (at least used to) only have access to the ID. The [docs](https://developer.hashicorp.com/terraform/tutorials/providers-plugin-framework/providers-plugin-framework-resource-import#implement-import-functionality) don't seem to suggest there's a change here.

Reply:
I see what you mean. Let me try to test API, if get by ID without namespace works for not `default` namespace.

Reply:
Interesting, that [import test](https://github.com/elastic/terraform-provider-elasticstack/blob/main/internal/kibana/synthetics/private_location/acc_test.go#L66) for `private_location` worked with `testacc` 

Does it answer on your question for `private_location` resource import or shall I test something else?

Reply:
Okey, tested APIs - private location works, but monitor. 

This works
```
POST  /s/testacc/api/synthetics/private_locations # creates a private location with id 97a1d865-5d52-4e52-838f-9af6a186d97e
GET  /api/synthetics/private_locations/97a1d865-5d52-4e52-838f-9af6a186d97e # return correct result

```

This doesn't work:
```
POST  /s/testacc/api/synthetics/monitors #creates a monitor with id 510bedd2-58a9-47aa-921b-bb9c2bc4ff34
GET  /api/synthetics/monitors/510bedd2-58a9-47aa-921b-bb9c2bc4ff34 # 404
```

Good catch, @tobio .

Last question from my side: is the import acceptance test, as in the example above enough or shall I test import somehow else?

Reply:
IMHO import test with `testacc` namespace is good enough. I was able to reproduce the issue moving http monitor to `testacc` 

```
Step 2/6 error running import: exit status 1

        Error: Failed to get monitor `eeae1afb-014e-475c-a826-799f829e9fe1`, namespace

        Monitor id eeae1afb-014e-475c-a826-799f829e9fe1 not found!%!(EXTRA string=Not
        Found)
```

@tobio , let me know if I need to add different way to verify imports.

Reply:
Done. I have migrated private location to composite id as well.